CONTRIBUTING.md

```diff
@@ -26,6 +26,12 @@ the guidelines we follow for how documents are formatted.
 
 Each PR must be signed per the following section.
 
+### Assigning and Owning work
+
+If you want to own and work on an issue, add a comment or “#dibs” it asking 
+about ownership. Contributors will then add the "Assigned" label with 
+“Assigned to: person@xxx”.
```

Comment:
A maintainer will then add the `Assigned` label and modify the first comment
in the issue to include `Assigned to: @person`.